components/base/Typography.tsx

```diff
@@ -136,6 +137,50 @@ export const listStyles = css`
   }
 `;
 
+// Table styles
+export const tableStyles = css`
```

Comment:
I believe the designs have tables taking up the full width of the content area whenever they're used. Is that something we could build in here? Or do you think there are some situations where it would be better only be the width of the content?